### ISSUE TITLE
harvest_send fails for python3

### ISSUE THREAD

**Issue by smithsp** [gafusion/harvest_client#4]
I gave a bare:
```python
harvest_send({'test':'Fail'}, table='test_harvest', verbose=True)
```
in OMFIT.   It gave the error:
```python
TypeError("a bytes-like object is required, not 'str'")
```
I think this is the cause of OMFIT usage appearing to drop after Jan 1, 2020.

**Comment by orso82** [comment]
closed with f6f8484

**Comment by smithsp** [comment]
Thanks.  I can confirm that this fixes the OMFIT stats uploading problem.
